src/test/kotlin/com/terraformation/backend/documentproducer/api/DocumentsControllerTest.kt

```diff
@@ -589,8 +604,10 @@ class DocumentsControllerTest : ControllerIntegrationTest() {
 
     @Test
     fun `returns not found error for nonexistent manifest`() {
+      val documentId = insertDocument()
```

Comment:
Out of curiosity, why does this need to insert a document explicitly rather than using the default `cannedDocumentId`?

Reply:
Wasn't needed after all. I put it in there because I was thinking it was returning a 404 for missing document since it isn't a "real" document, but it is in fact returning the nonexistent manifest error.

This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.

Reply:
This ended up staying in place as all the "canned" variables were ditched in favor of the predominant pattern in `DatabaseBackedTest`